include/mbedtls/error.h

```diff
@@ -52,9 +52,10 @@
  * For historical reasons, low-level error codes are divided in even and odd,
  * even codes were assigned first, and -1 is reserved for other errors.
  *
- * Low-level module errors (0x0002-0x007E, 0x0003-0x007F)
+ * Low-level module errors (0x0001-0x007E, 0x0003-0x007F)
```

Comment:
Shouldn't it rather be (even, odd)?
```suggestion
 * Low-level module errors (0x0002-0x007E, 0x0001-0x007F)
```

Reply:
Indeed, my bad.